translations/ui-circulation/de.json

```diff
@@ -444,5 +444,7 @@
     "settings.checkout.audioTheme": "Select audio-alerts theme",
     "settings.checkout.audioTheme.classic": "Classico",
     "settings.checkout.audioTheme.modern": "Moderna",
-    "settings.checkout.audioTheme.future": "Futura"
-}
\ No newline at end of file
+    "settings.checkout.audioTheme.future": "Futura",
+
+    "mod-circulation.noItemFoundForBarcode": "Es existiert kein Exemplar mit dem Barcode {itemBarcode}"
```

Comment:
@julianladisch  We should add translation only in file translations/ui-circulation/en.json in all other file translation will be added by "FOLIO Translations Bot" 
@zburke  Could you please confirm that i understood all correctly and we need add translation only in file  translations/ui-circulation/en.json?

Reply:
This translation is temporary and will be overwritten the next time when the translations from https://lokalise.com/ get imported.
I provide the temporary translation as an example that the developer of https://issues.folio.org/browse/UIREQ-603 can use for manual testing.